Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -568,6 +575,12 @@ class ArbitraryMipmapDetector
       return {{SRGBToLinear(p[0]), SRGBToLinear(p[1]), SRGBToLinear(p[2]), SRGBToLinear(p[3])}};
     }
 
+    static PixelRGBAu8 SampleLinear(const u8* src, const Shape& src_shape, u32 x, u32 y)
```

Comment:
If they're no longer used, the Sample() and LinearToSRGB() and SRGBToLinear() functions can be removed